### PR TITLE
Updating Rails Blog Post

### PR DESCRIPTION
**Pull request by Couragyn**

### CODE REVIEW

File: 2023/01/updating-rails.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- rails
+- ruby
+- update
+- gem
+---
+
+## Updating Rails
+Updating your app to the latest version is an important part of the development process. It may seem like a waste to invest time and money into it, but it can bring as much value as a new feature.
+
+One good thing about using a framework like Rails is that security features are baked in. This saves development time as the devloper doesn't have to re-create the wheel for logins, permissions, authentication, etc. There are many users of the framework who work together to can catch and patch vulnerabilies. Unfortunately this means if your app hasn't been updated in becomes more obvious what its weaknesses are. A black hat developer even has access to a [list of previous Rails version vulnerabilies.](https://www.cvedetails.com/vulnerability-list/vendor_id-12043/product_id-22569/Rubyonrails-Rails.html)
```
Comment (NicholasPiano) [inline 2023/01/updating-rails.md:15]: s/vulnerabilies/vulnerabilities/

File: 2023/01/updating-rails.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+Any of these updates can introduce bugs, but the biggest pain points will be the big version changes (4 to 5, 5 to 6, etc.)
+
+### Fixing Bundle Errors
+![Gemfile](/blog/2022/12/updating-rails/upgraded-gemfile.png)
+
+After changing the Rails and Ruby version in the `Gemfile`, run `bundle update rails`.
+
+![Bundle Error](/blog/2022/12/updating-rails/bundle-error.png)
+
+Some Gems won't be happy with the Rails update and will also need to be updated. Once this is done run `bundle update rails` again. If there are more errors fix them. If it's successfull `bundle install` can then be run. This will ensure all the gems are properly installed.
```
Comment (NicholasPiano) [inline 2023/01/updating-rails.md:42]: errors [comma] fix them
s/successfull/successful/g

**Comment by jonjensen** [comment]
@Couragyn Are you all done with this for now? I see your new commit but no reply to Nicholas.

**Comment by sethjensen1** [comment]
@Couragyn This is live!

**Pull request closed without merging**